File: DotNet/ModelTracKer/Models/MicroService.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ModelTracKer.Models
{
    [Table("microservice")]
    public class MicroService
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public string? Name { get; set; }

        public ICollection<opp_microservice> OppMicroservice { get; set; }
    }
}

File: Java/tracker/tracker/src/main/java/com/tacker/tracker/models/Microservice.java
package com.tacker.tracker.models;

import jakarta.persistence.*;
import lombok.*;

import java.util.Set;

@Entity
@Table(name = "microservice")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Microservice {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int Id;

    private String value;

    @OneToMany(mappedBy = "microservice")
    private Set<OppMicroservice> oppMicroserviceSet;

}
